examples/3d/3d_scene_pipelined.rs

```diff
@@ -33,9 +34,13 @@ struct Movable;
 /// set up a simple 3D scene
 fn setup(
     mut commands: Commands,
+    mut bloom_settings: ResMut<BloomSettings>,
     mut meshes: ResMut<Assets<Mesh>>,
     mut materials: ResMut<Assets<StandardMaterial>>,
 ) {
+    // configure bloom threshold
+    bloom_settings.threshold = 10.0;
+
     // ground plane
     commands.spawn_bundle(PbrBundle {
```

Comment:
We do have noticeable bloom flickering for moving lights. To an extent, it seems unavoidable and a natural part of the effect, but the way it surfaces here seems a bit unnatural. Curious to see if other engines have the issue (to this extent). 

https://user-images.githubusercontent.com/2694663/142297400-91c4fe27-8ad4-4ae1-92ec-fb3820655797.mp4

Reply:
The most "out of place" instance of this starts on the green sphere at around the 7 second mark.

Reply:
I think this kind of specular aliasing is a bit difficult to completely eliminate. Especially with very low roughness values (where all of the specular energy is concentrated into a tiny speck, smaller than a pixel). Also The bloom intensity seems very high in this scene, increasing the effect further.

I'm not eintirely sure what kind of curve this filter produces:  [bloom.wgsl:119-128](https://github.com/bevyengine/bevy/pull/2876/files#diff-a648fc450983c3cc6e66fbb39ea74a5f7705e21eb473327735f4fe2a680f0f00R119-R128)
But at a quick glance it seems to grow x², which will exaggerates the flickering of small, bright areas more.

As for possible solutions to reduce the effect (other than adjusting bloom parameters) I see limiting the pixel brightness (maybe also based on the neighbouring pixels) or temporal filtering.
Personally I think I'd change the high-pass filter and limit the specular aliasing by using less problematic meshes/materials, but that might not be a solution for everyone.